cranelift/codegen/src/isa/x64/inst/args.rs

```diff
@@ -1,3 +1,5 @@
+#![allow(dead_code)]
```

Comment:
Is this still needed or was this a temporary addition?

Reply:
or rather, if it's still needed, mind adding a comment as to why?

Reply:
Ah, this appears to have been left over -- we seem to have fixed the warning subsequently. Removed!